src/goto-programs/goto_inline.cpp

```diff
@@ -203,7 +203,6 @@ void goto_partial_inline(
       }
     }
   }
-
```

Comment:
:pick: This empty line removal looks unrelated to the changes this commit is supposed to be making. Presumably this is due to adding and removing different changes in this PR, but this formatting change should be cleaned up. Same applied to the line marked with a :dash: